### PR TITLE
Fixed User model to take email instead of username

### PR DESCRIPTION
**Pull request by AbsurdlyEloquent**
## Related Issues (include the `#`)
n/a

## Description of Changes Made
Reorganized views and paths into modules
Wrote custom user manager to take email instead of username
*Django does **not** make this easy*

### CODE REVIEW

**Review by iandemed — COMMENTED** [review]
In `back-end/developerverse_project/urls.py` is there supposed to be two `urls` properties?

**Comment by iandemed** [comment]
Also, make sure to add the model to the admin panel!

File: back-end/developerverse/urls/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from django.conf.urls import include
+from . import users
+
+urlpatterns = [
+    path('users/', include(users))
+]
```
Comment (Underwaterr) [inline back-end/developerverse/urls/urls.py]: Not necessarily opposed but unsure why we have a separate "urls" folder with two scripts in it. If the answer is modularity, this seems like an abstraction that should happen at the app-level (e.g. refactoring the "developerverse" app into a "user" app)

**Review by Underwaterr — CHANGES_REQUESTED** [review]

**Review by Underwaterr — CHANGES_REQUESTED** [review]
Close! But the app naming convention is that `Users` should be in lower case.

**Review by Underwaterr — DISMISSED** [review]

**Review by Underwaterr — APPROVED** [review]

**Pull request merged**